core/templates/pages/exploration-player-page/layout-directives/feedback-popup.component.html

```diff
@@ -18,7 +18,7 @@
     <button mat-button class="float-right oppia-exploration-feedback-submit-btn e2e-test-exploration-feedback-submit-btn"
             [ngClass]="{'oppia-feedback-popover-submit-btn-enabled': feedbackText}"
             (click)="saveFeedback()"
-            [disabled]="!feedbackText || feedbackText.length > MAX_REVIEW_MESSAGE_LENGTH">
+            [disabled]="!feedbackText || feedbackText.length > MAX_REVIEW_MESSAGE_LENGTH" type="button" class="btn btn-primary">
```

Comment:
```suggestion
            [disabled]="!feedbackText || feedbackText.length > MAX_REVIEW_MESSAGE_LENGTH" 
            type="button" 
            class="btn btn-primary">
```

Try to write attributes on different lines.